shared_model/validators/field_validator.cpp

```diff
@@ -284,7 +288,7 @@ namespace shared_model {
             "CreatedTime",
             {fmt::format(
                 "sent from future, timestamp: {}, now: {}", timestamp, now)});
-      } else if (now > kMaxDelay + timestamp) {
+      } else if (now > kMaxDelay.count() + timestamp) {
```

Comment:
@baziorek can you explain why this is necessary?

Reply:
> @baziorek can you explain why this is necessary?

@appetrosyan As I remember it was problem with type and conversion between types.
_______
Previously code used `auto`:
```
      static constexpr auto kMaxDelay =
          std::chrono::hours(24) / std::chrono::milliseconds(1);
```
but when I needed to separate variable declaration and definition I needed to have standard type:
```
      static std::chrono::milliseconds kMaxDelay;
```
And those types are not compatible anymore, so I can't do:
```
now > kMaxDelay + timestamp
```
- [ ] But I'll try to investigate this and answer You. Maybe I can remove `.count()`.

Reply:
No problem. I think you changed the type by accident. If this works, and is readable, I see no problem.

Reply:
After changing type into `time_t` using method `.count()` is not required.